notebooks/official/bigquery_ml/bqml-online-prediction.ipynb

```diff
@@ -32,18 +32,18 @@
         "<table align=\"left\">\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    ! bq rm -f --model $PROJECT_ID\:$BQ_DATASET_NAME\.$BQML_MODEL_NAME
</pre><p>Do you need to delete BQ datasets?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>

Reply:
Not sure if you mean -- "are you sure you need to delete BQ datasets"? Which in case, yes, if we're deleting other resources too.